src/scripts/content/ticktick.js

```diff
@@ -1,19 +1,25 @@
 'use strict';
 
-togglbutton.render('#task-detail-view:not(.toggl)', { observe: true }, function (
-  elem
-) {
-  const project = $('#project-setting input', elem).getAttribute('value');
+togglbutton.render('#task-detail-view:not(.toggl)', { observe: true }, function renderTickTick (elem) {
+  function getProject () {
+    const projectEl = elem.querySelector('.project-setting input');
+    return projectEl ? projectEl.value.trim() : '';
+  }
 
-  const text = function () {
-    return $('.task-title', elem).textContent;
-  };
+  function getDescription () {
+    const descriptionEl = elem.querySelector('#tasktitle div[contenteditable]');
+    return (descriptionEl ? descriptionEl.textContent : '').trim();
```

Comment:
Could we change this so it's consistent with L6 (`descriptionEl.textContent.trim()  : '');`)?

Reply:
Done ✅